### PR TITLE
MB-8801 add system error component

### PR DESCRIPTION
**Pull request by mlm55**
## Description

We want a general error that we can display on the Office and Customer apps when there are errors outside of the user’s control. This PR creates a SystemError component and adds it to storybook.

## Setup

```sh
make storybook
```
http://localhost:6006/?path=/story/components-system-error--system-error-component

## Code Review Verification Steps
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-8801) for this change

### CODE REVIEW

File: src/components/SystemError/index.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const SystemError = () => (
+  <div className="usa-alert--system-error">
```
Comment (Ronolibert) [inline src/components/SystemError/index.jsx:4]: Do you know where the docs are for this `usa-alert--system-error` class? I've looked on uswds but can't find its existence

**Review by Ronolibert — APPROVED** [review]
Just had a question for curiosity but otherwise looks good

File: src/components/SystemError/index.jsx (hunk 1 above)
Comment (mlm55) [reply src/components/SystemError/index.jsx:4]: src/shared/styles/_overrides.scss

File: src/components/SystemError/index.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    Something isn&apos;t working, but we&apos;re not sure what. Wait a minute and try again. If that doesn&apos;t fix
+    it, contact the Technical Help Desk and give them this code: <strong>[trace ID]</strong>.
```
Comment (YanZ777) [inline src/components/SystemError/index.jsx:6]: I think we want to be able to pass the trace id as props, but I am not sure if that should be handled in this story or in the one that actually hooks up the trace id.

**Review by YanZ777 — APPROVED** [review]
Approving as my question is not blocking, and I think putting the work in a following story will not significantly increase the scope,

**Review by cosburn — APPROVED** [review]
Looks good!

**Pull request merged**